examples/plot_mne_example.py

```diff
@@ -53,7 +53,7 @@
 # For the current example, we are going to sub-select only the EEG data,
 # and analyze it as continuous (non-epoched) data.
 #
-# Note that if you don't already have the data, the `data_path` command
+# Note that if you don't already have the data, the :meth:`~.sample.data_path` method
```

Comment:
this, I think, would need intersphinx connecting to MNE to link properly (right now it doesn't link properly). 
I think we probably don't need to add intersphinx to `MNE`, so you can drop the :meth: directive here.

Reply:
:meth:`~.sample.data_path` change to ``sample.data_path``